packages/build-tools/pkg/buildsys/parser.go

```diff
@@ -352,6 +354,40 @@ func task(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kw
 	return task, nil
 }
 
+func loadModule(thread *starlark.Thread, module string) (starlark.StringDict, error) {
```

Comment:
What are the additional string arguments this function takes? Probably should document this here (or in `tasks.star`).

Reply:
It's just one parameter and it's the module (filename) to load. I expected `loadModule(... module)` to be obvious enough or are you talking about the `load(..., "a", "b", "c")` part in the `*.star` files?
Those are the imported variables / symbols as [documented in the language spec](https://github.com/google/starlark-go/blob/7a1108eaa0/doc/spec.md#load-statements).

Reply:
Ah, that's the part I wasn't seeing. Maybe a quick inclusion of the relevant details in `tasks.star`'s docstring then, since I wouldn't think to look at the language spec for `load()`.

Reply:
I've added a comment about this. I think we should eventually reorganize that documentation (and update it) now that we have several files but I'm not yet sure how that would look.